src/main/java/io/jenkins/plugins/gitlabbranchsource/TriggerMRCommentTrait.java

```diff
@@ -48,14 +48,23 @@ public String getCommentBody() {
         return commentBody;
     }
 
+    /**
+     * Allow trigger a new build by trusted members only.
+     *
+     * @return true if allow trusted members only
```

Comment:
```suggestion
     * @return true if allow trusted members only
     * @since TODO
```

Reply:
This's a good suggestion in my view. I'm going to commit it. Please fix it if there's anyone who has other opinions.